server/application-data/application-data.json

```diff
@@ -1,19 +1,67 @@
 {
-    "administrator-credential-list":
-    [
-      {
-        "administrator-id" : "1",
-        "administrator-name" : "administrator",
-        "user-name" : "admin.user",
-        "authorization" : " *** Basic Ath Code *** ",
-        "allowed-methods" : "ALL"
-      },
-      {
-        "administrator-id" : "2",
-        "administrator-name" : "administrator",
-        "user-name" : "admin.user",
-        "authorization" : " *** Basic Ath Code *** ",
-        "allowed-methods" : "PUT"
-      }
-    ]
-  }
\ No newline at end of file
+  "administrator-credential-list": [
+    {
+      "administrator-id": "1",
+      "administrator-name": "administrator",
+      "user-name": "admin.user",
+      "authorization": " *** Basic Ath Code *** ",
+      "allowed-methods": "CONNECT"
+    },
+    {
+      "administrator-id": "2",
+      "administrator-name": "administrator",
+      "user-name": "admin.user",
+      "authorization": " *** Basic Ath Code *** ",
+      "allowed-methods": "DELETE"
+    },
+    {
+      "administrator-id": "3",
+      "administrator-name": "administrator",
+      "user-name": "admin.user",
+      "authorization": "Basic aXN3YXJ5YWEuc3ViYXNoY2hhbmRyYW46aXN3YXJ5YWE=",
```

Comment:
please remove the valid authorization code in github as it is confidential information